cmd/zapi/cmd/get/get.go

```diff
@@ -173,15 +174,23 @@ func parseExpr(spaceID api.SpaceID, expr string) (*api.SearchRequest, error) {
 }
 
 // parseExprWithChunk creates an api.WorkerRequest to be used with the client.
-func parseExprWithChunk(spaceID api.SpaceID, expr string, chunkInfo string) (*api.WorkerRequest, error) {
+func parseExprWithChunk(expr string, chunkPath string) (*api.WorkerRequest, error) {
 	// This is only for testing using the -chunk flag
-	searchRequest, err := parseExpr(spaceID, expr)
+	search, err := zql.ParseProc(expr)
+	if err != nil {
+		return nil, err
+	}
+	proc, err := json.Marshal(search)
 	if err != nil {
 		return nil, err
 	}
 	return &api.WorkerRequest{
-		SearchRequest: *searchRequest,
-		ChunkPaths:    []string{chunkInfo},
+		SearchRequest: api.SearchRequest{
+			Proc: proc,
+			Dir:  -1,
+		},
+		DataPath:   filepath.Join(filepath.Dir(chunkPath), "../.."),
```

Comment:
If the path for a chunk is an s3 uri, this will fail for windows. I would suggest using iosrc.URI for this.

Reply:
Also, is there any reason why the server can't be the one to derive this from the provided chunk paths? Can we get rid of this field?

Reply:
FYI the `ChunkPaths` field above are relative chunk paths, relative to the data path of the archive, so they're not redundant with the DataPath field.

Reply:
I might be missing something about iosrc, but I do not see a way to do the equivalent of filepath.Join with iosrc. I was modeling this code roughly on the pattern in archive/walk.go, so I thought using relative file paths was OK in this context.

BTW, I tried this:

	chunkURI, err := iosrc.ParseURI(chunkPath)
...
	datapath := chunkURI.AppendPath("../..").String()

To get the relative path using iosrc, but it didn't work.

Reply:
Actually given Al's comment that chunks will be relative, I think you should just do path.Join(path.Dir(chunkPath), "../..")

Reply:
ok